src/components/StaticInfo.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+
+import FullWidthContainer from './FullWidthContainer';
+import LargeText from './LargeText';
+
+import recycleFacts from '../data/recycling-facts.json';
+
+class StaticInfo extends Component {
+  insertRandomFact() {
+    const randomIndex = Math.floor(Math.random() * recycleFacts.length);
+    return recycleFacts[randomIndex];
+  }
+
+  render() {
+    return(
+      <FullWidthContainer flex={this.props.containerFlex}>
+        <LargeText
+          style={styles.largeText}
+          text={this.insertRandomFact()} />
+      </FullWidthContainer>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  largeText: {
+    padding: "5%"
```

Comment:
I have a strong bias against %s (besides 100% and 50%). Prefer pixels or rems. We should create a standard padding variable so it's easier to change later.

Reply:
Well this seems to be the standard for React Native is percentages and pixels, so REMs isn't an option.